study.md

```diff
@@ -93,15 +93,18 @@ Ruby:
 ```ruby
 array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
 
-# your answer here
+array.map {|x| x * x}
 ```
 
 JavaScript:
 
 ```javascript
 const array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
 
-// your answer here
+let square = function (x) {
+  return x * x
+}
+let newArray = array.map(square, x)
```

Comment:
This will cause an error since the optional second argument to `map` is expected to be a context for `this` during iteration.